src/fontra/client/core/glyph-svg.js

```diff
@@ -51,3 +51,19 @@ function formatCoordinate(x, y, numerator, denominator, dx, dy) {
   y = Math.round(y * numerator + dy) / denominator;
   return `${x},${y}`;
 }
+
+export function pathToSVG(instance, bounds) {
+  const path = new SVGPath2D();
+  instance.path.drawToPath2d(path);
+  const pathString = path.getPath();
+
+  const width = bounds["xMax"] - bounds["xMin"];
+  const height = bounds["yMax"] - bounds["yMin"];
+
+  const svgString = `<svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
The extra indentation in the SVG data should be removed: it's more important that the SVG looks correctly indented then our code.